bot.go

```diff
@@ -2,56 +2,67 @@ package main
 
 import (
 	"bytes"
-	"encoding/binary"
 	"fmt"
-	"github.com/boltdb/bolt"
-	"github.com/yuhanfang/riot/constants/region"
 	"image/png"
 	"log"
 	"os"
 	"os/signal"
 	"strings"
 	"syscall"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/pixelrazor/pixelbot/repository"
+	"github.com/yuhanfang/riot/constants/region"
 
 	"github.com/bwmarrin/discordgo"
 )
 
 var (
-	logger    *log.Logger
+	logger *log.Logger
+	// TODO: get this programatically?
+	// myID = "133651422154588161"
 	dmchannel = "460949603496230914" // test bot
 	//dmchannel = "404261686057631748" // real bot
+
+	repo repository.Implementation
 )
 
-func main() { /*
-		os.Mkdir("logs", os.ModeDir)
-		f, _ := os.Create("logs/" + time.Now().Format("2006-01-02_15-04-05") + ".log")
-		defer f.Close()
-		logger = log.New(f, "", log.LstdFlags)*/
-	logger = log.New(os.Stdout, "", log.LstdFlags)
+func main() {
+	logger = log.New(os.Stdout, "", log.LstdFlags) // TODO: logrus ?
 	// Load the API key
-	riotKey := os.Getenv("RIOT_API")
-	osuKey := os.Getenv("OSU_API")
-	discordKey := os.Getenv("DISCORD_API")
-	fmt.Println("Riot api key:", riotKey)
-	fmt.Println("Osu api key:", osuKey)
-	fmt.Println("Discord api key:", discordKey)
+	riotKey, ok := os.LookupEnv("RIOT_API")
+	if !ok {
+		logger.Fatalln("Missing RIOT_API in env")
+	}
+	osuKey, ok := os.LookupEnv("OSU_API")
+	if !ok {
+		logger.Fatalln("Missing OSU_API in env")
+	}
+	discordKey, ok := os.LookupEnv("DISCORD_API")
+	if !ok {
+		logger.Fatalln("Missing DISCORD_API in env")
+	}
+
 	discord, err := discordgo.New("Bot " + discordKey)
 	if err != nil {
 		fmt.Println("Error making discordbot object:", err)
```

Comment:
Since this is the main function and all that happens after an error is a return, shouldn't this be logger.Fatalln() ?

Reply:
ah i overlooked this! there's a handful of areas i didn't catch when switching from fmt to a logger a while back. I need to do a general cleanup anyways, so that'll be in a different PR